# Pose of a tower
std_msgs/Header header
Tower[] towers